pkg/api/api.go

```diff
@@ -48,3 +49,16 @@ func New(tags *tags.Tags, storer storage.Storer, corsAllowedOrigins []string, lo
 
 	return s
 }
+
+const (
+	SwarmPinHeader = "Swarm-Pin"
+	TagHeaderUid   = "swarm-tag-uid"
```

Comment:
As we changed the format of headers names to Pascal kebeb case, this value can follow that.

Reply:
yeah, i didn't change that because of the open PR from @mortelli that has extensive changes to the tag api. it will introduce more conflicts. That's also why I did not change the variable name (should be `SwarmTagUidHeader`)